diff --git a/node_modules/@revenuecat/purchases-capacitor/android/build.gradle b/node_modules/@revenuecat/purchases-capacitor/android/build.gradle
index 5ca7b5f..17ce079 100644
--- a/node_modules/@revenuecat/purchases-capacitor/android/build.gradle
+++ b/node_modules/@revenuecat/purchases-capacitor/android/build.gradle
@@ -34,7 +34,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
